modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Inner/Run/Run.tsx

```diff
@@ -41,13 +51,28 @@ const Runs = () => {
 					actions,
 					columns: [
 						{
+							clickable: true,
 							key: 'number',
 							render: (number) =>
 								number?.toString().padStart(2, '0'),
 							value: i18n.translate('run'),
 						},
 						{
+							clickable: true,
 							key: 'applicationServer',
+							render: (applicationServer, item) => {
+								return (
+									<div
```

Comment:
Remove the return and use ( parameters plz

```suggestion
									<div
```